### PR TITLE
fix(work_order): Add item name in Work Order

### PR DESCRIPTION
**Pull request by Alchez**
**Problem:**

The manufactured item's name is not stored in the Work Order itself, just the item description. Although it's generally useful info to have on the document, one use case is that it can be used as the WO's title field, if required.

**Patch:**

image

**Form View:**

image

### CODE REVIEW

File: erpnext/patches/v12_0/add_item_name_in_work_orders.py [hunk 1]
```diff
@@ -0,0 +1,13 @@
+import frappe
+
+
+def execute():
+	frappe.reload_doc("manufacturing", "doctype", "work_order")
+
+	for wo in frappe.get_all("Work Order"):
```
Comment (rmehta) [inline erpnext/patches/v12_0/add_item_name_in_work_orders.py]: this can be fixed in one `UPDATE` query

**Review by rmehta — DISMISSED** [review]

File: erpnext/patches/v12_0/add_item_name_in_work_orders.py (hunk 1 above)
Comment (Alchez) [reply erpnext/patches/v12_0/add_item_name_in_work_orders.py]: Updated with `UPDATE` :)

**Comment by sagarvora** [comment]
Pls fix merge conflict.

**Comment by Anurag810** [comment]
Please resolve merge conflicts

**Pull request merged**